src/OpenIddict.Core/OpenIddictModule.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Microsoft.AspNet.Builder;
+
+namespace OpenIddict {
+    /// <summary>
+    /// Defines an OpenIddict module.
+    /// </summary>
+    public class OpenIddictModule {
+        /// <summary>
+        /// Gets or sets the position of the module in the ASP.NET pipeline.
+        /// </summary>
+        public int Position { get; set; }
```

Comment:
Nothing particular: we just assume the order between the modules sharing the same position doesn't matter and that they can be registered in any order between them.